### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by jbscoggi** [mutationpp/Mutationpp#175] [Ideas]
Hi All, as we discussed in the developers meeting, it is way past time to get a website up and running.  Let's use this discussion to try and come to a consensus about what the website should look like and contain.  Please feel free to also post links to other websites that could be a good model for M++.

**Comment by jbscoggi** [comment]
I'll get us started with some ideas of things that the website should have:

1. Obviously a link to the main repo.
2. Host the most recent documentation, automatically built after each PR merge
3. List of publications that make use of M++.  Doesn't have to just be papers of the developers, could also be our users who would like to advertise their work.
4. A show and tell of neat results that users provide us.

I think some good models could be [SU2](https://su2code.github.io/) and [Cantera](https://cantera.org/).

**Comment by rdbisme** [comment]
I personally would use Sphinx with MySt plugin + a theme similar to Gitlab website like [Furo](https://sphinx-themes.org/sample-sites/furo/)
